include/node_kernels/TKESSTNodeKernel.h

```diff
@@ -48,14 +47,15 @@ class TKESSTNodeKernel : public NGPNodeKernel<TKESSTNodeKernel>
   stk::mesh::NgpField<double> density_;
   stk::mesh::NgpField<double> tvisc_;
   stk::mesh::NgpField<double> dudx_;
+  stk::mesh::NgpField<double> coords_;
```

Comment:
going to remove these, they shouldn't be there as they are from previous debugging